doc/qa.rst

```diff
@@ -50,7 +54,7 @@ Wavelength Tilts QA
 There are generally a series of PNG files describing the analysis of the
 tilts of the arc lines.
 
-See :doc:`master_tilts` for a discussion of this QA.
+See :doc:`calibrations/master_tilts` for a discussion of this QA.
 
 
 Exposure QA
```

Comment:
It would be good to add under the Exposure QAs, the Object Finding QA that shows the object profile collapsed along the spectral direction. You could use the same image that I put here https://pypeit.readthedocs.io/en/develop/mosfire_howto.html#object-finding

Reply:
Done!  I also added one for the wavelength calibration.